tests/Provider/PoolTest.php

```diff
@@ -73,6 +76,16 @@ public function testGetInvalidProviderName()
         $mediaPool->getProvider('provider_c');
     }
 
+    public function testPoolAlias()
+    {
+        $container = new ContainerBuilder();
+        $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../../src/Resources/config'));
+        $loader->load('provider.xml');
+        $pool = $container->get('sonata.media.pool');
+        $alias = $container->get('%sonata.media.pool.class%');
+        $this->assertEquals($pool, $alias);
+    }
```

Comment:
Sorry I was unclear (I was on my phone at that time), I was thinking about this test: https://github.com/sonata-project/SonataMediaBundle/blob/3.x/tests/DependencyInjection/SonataMediaExtensionTest.php

Reply:
Done.